README.md

```diff
@@ -72,7 +72,7 @@ on this plugin. Cache can also be configured optionally.
 ossIndexAudit {
     username = 'email' // if not provided, an anonymous query will be made
     password = 'pass'
-    allConfigurations = true // if true includes the dependencies in all resolvable configurations. By default is false, meaning only 'compileClasspath' and 'releaseCompileClasspath' are considered
+    allConfigurations = true // if true includes the dependencies in all resolvable configurations. By default is false, meaning only 'compileClasspath', 'runtimeClasspath', 'releaseCompileClasspath' and 'releaseRuntimeClasspath' are considered
```

Comment:
Gradle allows to create custom configurations, so to avoid extra explanations or links to explain `compile` vs `implementation` vs `api` I decided to mention the default configurations. Legacy configurations are not mention for simplicity (hopefully only a few projects are using them nowadays).